src/CodeStyle/Core/CodeFixes/FixAllContextHelper.cs

```diff
@@ -74,12 +74,13 @@ internal static class FixAllContextHelper
             }
 
             return await GetDocumentDiagnosticsToFixAsync(
-                allDiagnostics, projectsToFix, fixAllContext.CancellationToken).ConfigureAwait(false);
+                allDiagnostics, projectsToFix, isGeneratedCode, fixAllContext.CancellationToken).ConfigureAwait(false);
         }
 
         private static async Task<ImmutableDictionary<Document, ImmutableArray<Diagnostic>>> GetDocumentDiagnosticsToFixAsync(
             ImmutableArray<Diagnostic> diagnostics,
             ImmutableArray<Project> projects,
+            Func<Document, CancellationToken, bool> isGeneratedCode,
```

Comment:
We have had multiple discussions in past about whether generated code should be skipped by default in fix all framework or the fixer should write a custom fixall provider to do such filtering and haven’t come to any consensus. I will try to find old issues and discussions around it. This most certainly needs a design discussion. I am also not sure how this bug meets the Dev16.0 bar currently. Can you clarify how this is a regression from 15.0 given FixAll framework never skipped generated files?

Reply:
➡️ The regression is described in #33507.